### ISSUE TITLE
Flutter 3.0 support

### ISSUE THREAD

**Issue by aytunch** [ConnectyCube/connectycube-flutter-samples#210]
I just upgraded to the latest Flutter version and when I try to use the latest
`connectycube_sdk: ^2.1.1`

I get a dependency error.

> Because no versions of connectycube_sdk match >2.1.1 <3.0.0 and connectycube_sdk 2.1.1 depends on xmpp_stone 0.4.3-dev.1, connectycube_sdk ^2.1.1 requires xmpp_stone 0.4.3-dev.1.

**Comment by aytunch** [comment]
@TatankaConCube 
After more investigation it seems this happens when we use latest auto_router package along with CC
Auto route wants xml version 6.0.0 
However xmpp_stone which is used by CC uses xml version 5.1.0

I could not find the xmpp_stone package to open an issue. is it maintained by you people?
In the worst case can you fork the xmpp package so we can use continue using it?


```
Because no versions of connectycube_sdk match >2.1.1 <3.0.0 and connectycube_sdk 2.1.1 depends on xmpp_stone 0.4.3-dev.1, connectycube_sdk ^2.1.1 requires xmpp_stone 0.4.3-dev.1.
And because xmpp_stone 0.4.3-dev.1 depends on xml ^5.1.0 and auto_route_generator >=4.2.0 depends on xml ^6.0.1, connectycube_sdk ^2.1.1 is incompatible with auto_route_generator >=4.2.0.
So, because bare_bones depends on both connectycube_sdk ^2.1.1 and auto_route_generator ^4.2.0, version solving failed.
pub get failed (1; So, because bare_bones depends on both connectycube_sdk ^2.1.1 and auto_route_generator ^4.2.0, version solving failed.)
exit code 1
```

**Comment by TatankaConCube** [comment]
@aytunch the `xmpp_stone` is an open-source [lib](https://github.com/vukoye/xmpp_dart). The version `0.4.3-dev.1` was prepared by me with some features required for Connectycube SDK. We will try to plan to investigate and fix your issue next week.

**Comment by aytunch** [comment]
@TatankaConCube thanks

As a temporary workaround I am using this and at least can continue debugging

```
dependency_overrides:
  xml: ^5.1.0
```

**Comment by TatankaConCube** [comment]
@aytunch I made the PR with the required feature, you can ping @vucoye too to speed up merging it

**Comment by aytunch** [comment]
@TatankaConCube since your PR is merged, when can we expect to see the new CC version?
Thanks

**Comment by TatankaConCube** [comment]
@aytunch today we have released version 2.2.3 with required updates
